index.js

```diff
@@ -78,33 +78,23 @@ const unique = function(array) {
   return true
 }
 
-const isMultipleOf = function(name, multipleOf) {
-  let res
-  const factor =
-    (multipleOf | 0) !== multipleOf
-      ? Math.pow(
-          10,
-          multipleOf
-            .toString()
-            .split('.')
-            .pop().length
-        )
-      : 1
-  if (factor > 1) {
-    const factorName =
-      (name | 0) !== name
-        ? Math.pow(
-            10,
-            name
-              .toString()
-              .split('.')
-              .pop().length
-          )
-        : 1
-    if (factorName > factor) res = true
-    else res = Math.round(factor * name) % (factor * multipleOf)
-  } else res = name % multipleOf
-  return !res
+function digitsAfterDot(number) {
+  if ((number | 0) === number) return 0
+  return String(number).split('.').pop().length
+}
```

Comment:
`digitsAfterDot` needs to either be declared in the scope like `isMultipleOf` is declared, or declared inside `isMultipleOf`, otherwise you get reference errors when using `toModule`:
```
> validate = eval(require('.')({ multipleOf: 'bar' }).toModule())
[Function: validate]
> validate(2)
Thrown:
ReferenceError: digitsAfterDot is not defined
    at isMultipleOf (eval at <anonymous> (repl:1:12), <anonymous>:8:18)
    at validate (eval at <anonymous> (repl:1:12), <anonymous>:21:12)
```

Reply:
Oh. My bad.
We probably need a test for that.

Reply:
Fixed, thanks! No test though.